django/db/migrations/operations/models.py

```diff
@@ -1313,3 +1319,15 @@ def describe(self):
     @property
     def migration_name_fragment(self):
         return f"alter_{self.model_name_lower}_{self.constraint.name.lower()}"
+
+    def reduce(self, operation, app_label):
+        if (
+            (
+                isinstance(operation, AlterConstraint)
+                or isinstance(operation, RemoveConstraint)
+            )
```

Comment:
I usually just provide a tuple, like:
```suggestion
            isinstance(operation, (AlterConstraint, RemoveConstraint))
```